tests/wpt/metadata/html/semantics/forms/textfieldselection/selection-start-end.html.ini

```diff
@@ -1,5 +1,6 @@
 [selection-start-end.html]
   type: testharness
+  expected: TIMEOUT
```

Comment:
Why does this timeout?

Reply:
Possibly a false-positive from running the tests locally on my machine. Will revert.